src/components/Cards/CardsCreationForm.tsx

```diff
@@ -1,33 +1,96 @@
 import * as React from "react";
+import styles from "./creationForm.module.scss";
 
 interface CreationForm {
   onSubmit: (item: any) => void;
 }
 
+interface Validation {
+  gender?: string;
+  title?: string;
+  price?: string;
+  imageUrl?: string;
+  ok?: boolean;
+}
+
 const CardsCreationForm: React.FC<CreationForm> = ({ onSubmit }) => {
   const [gender, setGender] = React.useState<string>('');
   const [title, setTitle] = React.useState<string>('');
   const [price, setPrice] = React.useState<string>('');
   const [imageUrl, setImageUrl] = React.useState<string>('');
+  const [error, setError] = React.useState<Validation>({});
+  const genderRef = React.useRef<HTMLLabelElement>();
+  const titleRef = React.useRef<HTMLLabelElement>();
+  const priceRef = React.useRef<HTMLLabelElement>();
+  const imageUrlRef = React.useRef<HTMLLabelElement>();
+
+  const scrollIfNeeded = (validation: Validation) => {
+    let view ;
+    if(validation.gender) {
+      view = genderRef.current;
+    } else if (validation.title) {
+      view = titleRef.current;
+    } else if (validation.price) {
+      view = priceRef.current;
+    } else if (validation.imageUrl) {
+      view = imageUrlRef.current;
+    }
+    view?.scrollIntoView();
+  }
+
+
+  const validateInputs = () => {
```

Comment:
```suggestion
  const isEmpty = (item: string): string => item ? "" : "this field is required";

  const validateInputs = (): Validation => {
    return {
      gender: isEmpty(gender),
      price: isEmpty(price),
      title: isEmpty(title),
      imageUrl: isEmpty(imageUrl),
      ok: !!(gender && title && price && imageUrl)
    };
  }
```

Maybe that would be a little better.